### PR TITLE
feat(PER-32): agregar nuevos filtros al modulo perinatal

### PR DESCRIPTION
**Pull request by MarianoCampetella**
### Requerimiento
https://proyectos.andes.gob.ar/browse/PER-32

### Funcionalidad desarrollada 
1. Se agregaron dos nuevos filtros para buscar a un paciente por organizacion y/o por profesional

### UserStories llegó a completarse
- [X] Si
- [ ] No

### Requiere actualizaciones en la base de datos
- [ ] Si
- [X] No

### CODE REVIEW

**Comment by liquid36** [comment]
USUARIO: mcampetella  
BUILD NUMBER: [4597](http://condorpiedra.andes.gob.ar:85/4597/mochawesome.html)
CYPRESS RUN: [3608](https://dashboard.cypress.io/projects/xr7gft/runs/3608/specs)
TEST START: 2021-07-16T15:56:40.119Z
TOTAL: 399
SUCCESS: 398
FAIL: 0
SKIPPED: 1

**Review by negro89 — APPROVED** [review]

File: modules/perinatal/carnet-perinatal.routes.ts [hunk 1]
```diff
@@ -21,6 +21,15 @@ class CarnetPerinatalResource extends ResourceBase {
...
         },
+        organizacion: MongoQuery.matchString.withField('controles.organizacion.nombre'),
+        profesional: (valor) => {
+            return {
+                $or: [
+                    { 'controles.profesional.nombre': MongoQuery.partialString(valor[0]) },
+                    { 'controles.profesional.apellido': MongoQuery.partialString(valor[1]) }
+                ]
+            };
+        }
```
Comment (plammel) [inline modules/perinatal/carnet-perinatal.routes.ts:32]: La búsqueda de profesional debería ser por match de ObjectId de profesional

**Review by plammel — CHANGES_REQUESTED** [review]

File: modules/perinatal/carnet-perinatal.routes.ts [hunk 2]
```diff
@@ -21,6 +21,8 @@ class CarnetPerinatalResource extends ResourceBase {
                 ]
             };
         },
+        organizacion: MongoQuery.matchString.withField('controles.organizacion.nombre'),
+        profesional: MongoQuery.matchString.withField('controles.profesional.id'),
```
Comment (plammel) [inline modules/perinatal/carnet-perinatal.routes.ts:25]: Siendo que el id de profesional de control se guarda como tipo ObjectId la menara correcta de matchear sería:
```suggestion
        profesional: {
            field: 'controles.profesional.id',
            fn: (value) => mongoose.Types.ObjectId(value)
        }
```

**Review by plammel — CHANGES_REQUESTED** [review]

**Comment by liquid36** [comment]
USUARIO: mcampetella  
BUILD NUMBER: [4634](http://condorpiedra.andes.gob.ar:85/4634/mochawesome.html)
CYPRESS RUN: [3643](https://dashboard.cypress.io/projects/xr7gft/runs/3643/specs)
TEST START: 2021-07-21T11:48:40.681Z
TOTAL: 400
SUCCESS: 399
FAIL: 0
SKIPPED: 1

**Comment by liquid36** [comment]
USUARIO: mcampetella  
BUILD NUMBER: [4636](http://condorpiedra.andes.gob.ar:85/4636/mochawesome.html)
CYPRESS RUN: [3645](https://dashboard.cypress.io/projects/xr7gft/runs/3645/specs)
TEST START: 2021-07-21T12:16:35.561Z
TOTAL: 400
SUCCESS: 399
FAIL: 0
SKIPPED: 1

**Comment by liquid36** [comment]
USUARIO: mcampetella  
BUILD NUMBER: [4650](http://condorpiedra.andes.gob.ar:85/4650/mochawesome.html)
CYPRESS RUN: [3656](https://dashboard.cypress.io/projects/xr7gft/runs/3656/specs)
TEST START: 2021-07-22T11:40:31.005Z
TOTAL: 359
SUCCESS: 345
FAIL: 13
SKIPPED: 1

**Comment by liquid36** [comment]
USUARIO: mcampetella  
BUILD NUMBER: [4652](http://condorpiedra.andes.gob.ar:85/4652/mochawesome.html)
CYPRESS RUN: [3658](https://dashboard.cypress.io/projects/xr7gft/runs/3658/specs)
TEST START: 2021-07-22T12:22:34.936Z
TOTAL: 334
SUCCESS: 320
FAIL: 14
SKIPPED: 0

**Comment by liquid36** [comment]
USUARIO: mcampetella  
BUILD NUMBER: [4656](http://condorpiedra.andes.gob.ar:85/4656/mochawesome.html)
CYPRESS RUN: [3660](https://dashboard.cypress.io/projects/xr7gft/runs/3660/specs)
TEST START: 2021-07-22T13:12:53.844Z
TOTAL: 332
SUCCESS: 318
FAIL: 13
SKIPPED: 1

**Comment by liquid36** [comment]
USUARIO: mcampetella  
BUILD NUMBER: [4669](http://condorpiedra.andes.gob.ar:85/4669/mochawesome.html)
CYPRESS RUN: [3672](https://dashboard.cypress.io/projects/xr7gft/runs/3672/specs)
TEST START: 2021-07-22T16:28:29.348Z
TOTAL: 400
SUCCESS: 399
FAIL: 0
SKIPPED: 1

File: modules/perinatal/carnet-perinatal.routes.ts [hunk 3]
```diff
@@ -21,6 +22,11 @@ class CarnetPerinatalResource extends ResourceBase {
                 ]
             };
         },
+        organizacion: MongoQuery.matchString.withField('controles.organizacion.nombre'),
```
Comment (plammel) [inline modules/perinatal/carnet-perinatal.routes.ts:25]: Así como en profesional, el match de organización tiene que ser por id, en lugar de nombre.

**Review by plammel — CHANGES_REQUESTED** [review]

**Comment by liquid36** [comment]
USUARIO: mcampetella  
BUILD NUMBER: [4704](http://condorpiedra.andes.gob.ar:85/4704/mochawesome.html)
CYPRESS RUN: [3704](https://dashboard.cypress.io/projects/xr7gft/runs/3704/specs)
TEST START: 2021-07-26T16:59:41.724Z
TOTAL: 400
SUCCESS: 398
FAIL: 1
SKIPPED: 1

**Comment by liquid36** [comment]
USUARIO: cramos  
BUILD NUMBER: [4708](http://condorpiedra.andes.gob.ar:85/4708/mochawesome.html)
CYPRESS RUN: [3707](https://dashboard.cypress.io/projects/xr7gft/runs/3707/specs)
TEST START: 2021-07-26T17:37:50.958Z
TOTAL: 336
SUCCESS: 322
FAIL: 13
SKIPPED: 1

**Review by plammel — APPROVED** [review]

**Comment by liquid36** [comment]
USUARIO: mcampetella  
BUILD NUMBER: [4713](http://condorpiedra.andes.gob.ar:85/4713/mochawesome.html)
CYPRESS RUN: [3712](https://dashboard.cypress.io/projects/xr7gft/runs/3712/specs)
TEST START: 2021-07-26T19:07:23.025Z
TOTAL: 400
SUCCESS: 398
FAIL: 1
SKIPPED: 1

**Comment by liquid36** [comment]
USUARIO: cramos  
BUILD NUMBER: [4720](http://condorpiedra.andes.gob.ar:85/4720/mochawesome.html)
CYPRESS RUN: [3718](https://dashboard.cypress.io/projects/xr7gft/runs/3718/specs)
TEST START: 2021-07-27T11:37:21.337Z
TOTAL: 400
SUCCESS: 398
FAIL: 1
SKIPPED: 1

**Comment by liquid36** [comment]
USUARIO: mcampetella  
BUILD NUMBER: [4721](http://condorpiedra.andes.gob.ar:85/4721/mochawesome.html)
CYPRESS RUN: [3719](https://dashboard.cypress.io/projects/xr7gft/runs/3719/specs)
TEST START: 2021-07-27T11:47:28.039Z
TOTAL: 382
SUCCESS: 380
FAIL: 1
SKIPPED: 1

**Comment by liquid36** [comment]
USUARIO: mcampetella  
BUILD NUMBER: [4722](http://condorpiedra.andes.gob.ar:85/4722/mochawesome.html)
CYPRESS RUN: [3720](https://dashboard.cypress.io/projects/xr7gft/runs/3720/specs)
TEST START: 2021-07-27T11:57:59.137Z
TOTAL: 348
SUCCESS: 333
FAIL: 14
SKIPPED: 1

**Comment by liquid36** [comment]
USUARIO: cramos  
BUILD NUMBER: [4724](http://condorpiedra.andes.gob.ar:85/4724/mochawesome.html)
CYPRESS RUN: [3721](https://dashboard.cypress.io/projects/xr7gft/runs/3721/specs)
TEST START: 2021-07-27T12:32:28.543Z
TOTAL: 400
SUCCESS: 399
FAIL: 0
SKIPPED: 1

**Review by MCele — APPROVED** [review]

**Pull request merged**